src/Email/Body.php

```diff
@@ -19,20 +19,11 @@ class Body
 {
 	use Properties;
 
-	/**
-	 * @var string
-	 */
-	protected $html;
-
-	/**
-	 * @var string
-	 */
-	protected $text;
+	protected string|null $html = null;
+	protected string|null $text = null;
 
 	/**
 	 * Email body constructor
-	 *
-	 * @param array $props
 	 */
 	public function __construct(array $props = [])
```

Comment:
Wouldn't it be a good chance to refactor the constructor and get rid of the Properties trait, setText and setBody in one go? 

```php 
public function __construct(array $props = []) 
{
  $this->html = $props['html'] ?? null,
  $this->text = $props['text'] ?? null,
}
```